spec/base_spec.rb

```diff
@@ -466,6 +511,38 @@ def fixup_aggregate_schema(loaded_version)
         expect(queried_model.my_other_string).to eq("Hello")
       end
 
+      context "with inheritance" do
```

Comment:
For both tests in this context, I'd suggest verifying the query response models' types as well as the counts of models returned.